checkov/common/output/csv.py

```diff
@@ -51,7 +51,7 @@ def __init__(self) -> None:
     def add_report(self, report: Report, git_org: str, git_repository: str) -> None:
         if report.check_type in (CheckType.SCA_PACKAGE, CheckType.SCA_IMAGE):
             for record in itertools.chain(report.failed_checks, report.passed_checks, report.skipped_checks):
-                if record.check_name == SCA_PACKAGE_SCAN_CHECK_NAME:
+                if record.check_name in [SCA_IMAGE_SCAN_CHECK_NAME, SCA_PACKAGE_SCAN_CHECK_NAME]:
```

Comment:
```suggestion
                if record.check_name == SCA_PACKAGE_SCAN_CHECK_NAME:
```
it turns put you don't need to change anything here (my mistake)
it seems that there are no records that have check name of 'SCA_IMAGE_SCAN_CHECK_NAME'
if you will take a look at sca_image runner, when creating a cve record, it also have check_name of SCA_PACKAGE_SCAN_CHECK_NAME